### PR TITLE
feat(): select input

### PR DESCRIPTION
**Pull request by JordanBonitatis**

### CODE REVIEW

File: src/components/inputs/Money.js [hunk 1]
```diff
@@ -2,34 +2,35 @@ import React from 'react';
 import CurrencyTextField from '@unicef/material-ui-currency-textfield'
 
 export default class MoneyInput extends React.Component {        
-    // TODO calculated fields don't update
```
Comment (JordanBonitatis) [inline src/components/inputs/Money.js]: nothing changed here - just updated it from 4-spaced tabs to 2-spaced tabs

File: src/equations/endurance.js [hunk 2]
```diff
@@ -32,9 +32,12 @@ export default class Config {
     label: "Years Before Broke",
     type: "text"
   }
-  static equation = ({principal, burnAmount, totalInterest, deposits} = {}) => {
+  static equation = ({principal, burnAmount, burnPeriod, totalInterest, deposits} = {}) => {
```
Comment (JordanBonitatis) [inline src/equations/endurance.js]: i wanted to see the select actually modify the result. i noticed that this equation isn't even using all of its params. is that intentional?

File: src/components/Equation.js [hunk 3]
```diff
@@ -51,6 +52,18 @@ export default class Equation extends React.Component {
...
+          <SelectInput
+            id={param}
+            label={paramConfig.label}
+            key={param}
+            fieldKey={param}
+            options={paramConfig.options}
+            value={this.state[param]}    
+            onChange={this.handleChange}
+          />
+        )
```
Comment (rs-trevor) [inline src/components/Equation.js]: OFF TOPIC: This is making me think I'm going to change that PR to be a switch that returns a Jsx tag, which. you will in turn pass values to? what do you think?   A Tag to make tags seems a bit weird.   So this method will look like (pseudo code):

```
const Field = switch(paramConfig.type) {
case "currency": Currency
case "select": SelectInput
...
}

attribs.push(<Field id={param}  ... )
```

Does that make sense?

File: src/equations/endurance.js (hunk 2 above)
Comment (rs-trevor) [reply src/equations/endurance.js]: endurance is an interesting equation if you work in all these args, but I really added it as a bullshit equation to get us up and running.  It's not entirely implemented.  Compounding these numbers wasn't the easiest thing to do.  We could even delete endurance if we want.

File: src/equations/endurance.js [hunk 4]
```diff
@@ -32,9 +32,12 @@ export default class Config {
...
     totalInterest = 0.0;
     deposits = 0.0;
+    if (burnPeriod === 30) {
+      burnAmount = burnAmount * 12;
+    }
```
Comment (rs-trevor) [inline src/equations/endurance.js]: hate this if statement.  This is trying to do something with months?

Comment (JordanBonitatis) [reply src/equations/endurance.js]: just wanted to make sure making a selection with the new input was updating the result. we can fix this later

File: src/components/Equation.js (hunk 3 above)
Comment (JordanBonitatis) [reply src/components/Equation.js]: Yeah - did you see my feedback on that PR? We can chat through it if you like

**Comment by trevorgrayson** [comment]
i have some errors when i try to run this. not sure if they are prexisting. scoping it out.

**Pull request merged**